pybaseball/batting_and_pitching_splits

```diff
@@ -0,0 +1,139 @@
+    """
+    Get the urls from baseball reference for split data for each player id. Returns the set of URLs 
+    for the playerid and splits provided. 
+    Use PitchOrBat='b' for batting splits or , PitchOrBat='p' for pitching splits
+    To get all batting split data tables, pass in 
+    splits=['power', 'plato', 'gbfb', 'clutc', 'lever', 'count', 'hmvis', 'month', 'outcb', 'stsub', 'half', 'leado', 'lineu', 'outs', 'bases', 'times', 'hitlo', 'traj', 'oppon', 'stad', 'total', 'site']
+        total - Season/Career Totals, last 7 days, last 14 days, last 28 days, last 365 days
+        power - vs Power/Finesse Pitchers splits
+        plato - platoon splits (e.g. vs RHP, vs LHP)
+        gbfb - vs Ground Ball/Fly Ball Pitchers
+        clutc - clutch stats (e.g. 2 outs RISP, Late & Close, Tie Game)
+        lever - Leverage splits
+        count - Count/Ball-Strikes
+        hmvis - home or away splits
+        month - month splits
+        outcb - game outcome for team splits (in wins, in losses)
+        stsub - starter or substitue splits
+        half - first or second half splits
+        leado - leading off the inning splits 
+        lineu - batting order position splits
+        outs - number of outs in inning  
+        bases - bases occupied 
+        times - times facing opponent in game
+        hitlo - hit location splits
+        traj - Hit trajectory (e.g. Ground balls, flyballs, line drives, bunts)
+        oppon - opponent splits
+        stad - game conditions
+        site - ballpark
+
+
+    To get all pitching splits, pass in
+    ['total',  'plato', 'hmvis', 'half', 'month', 'outco', 'sprel', 'rs', 'lineu', 'tkswg', 'count', 'leado', 'defpo', 'outs', 'bases', 'clutc', 'lever', 'innng', 'times', 'pitco', 'dr', 'hitlo', 'traj', 'oppon', 'stad', 'site', 'catch', 'ump'] 
+        total - Season/Career Totals, last 7 days, last 14 days, last 28 days, last 365 days
+        plato - platoon splits (e.g. vs RHP, vs LHP)
+        hmvis - home or away splits
+        half - first or second half splits
+        month - month splits
+        outco - game outcome for team splits (in wins, in losses)
+        sprel - Pitching Role (as starter, as reliever)
+        rs - Run Support splits
+        lineu - Batting order positions
+        tkswg - Swung or took first pitch of PA
+        count - Count/Balls-Strikes
+        leado - Leading off Inning
+        defpo - Defensive position (vs pitcher, vs non-pitcher)
+        outs - Number of outs in inning
+        bases - Bases occupied
+        clutc - Clutch Stats
+        lever - Leverage
+        innng - By inning
+        times - Times facing opponent in Game
+        pitco - Pitch count
+        dr - Days of Rest
+        hitlo - Hit Location
+        traj - Hit Trajectory
+        oppon - opponent splits
+        stad - game conditions  
+        site - ballpark
+        catch - By Catcher
+        ump - By Umpire
+
+    """
```

Comment:
I would reduce the docstring to more limited usage stuff and move the full documentation to its own file (see our standard documentation format)

Reply:
I reduced the docstring, but haven't created the documentation file yet. Will work on that in the coming days.